app/src/context/Exchange.tsx

```diff
@@ -162,7 +162,11 @@ export const ExchangeProvider: React.FC<{
       }
     } catch (error: any) {
       console.log(error);
-      Alert.alert(error.response.data);
+      if (error.response.data === "Username or password is incorrect") {
+        Alert.alert("Invalid password");
+      } else {
+        Alert.alert(error.response.data);
+      }
```

Comment:
Reduce the complexity by removing the else.

```suggestion
      }
      Alert.alert(error.response.data);
```

Reply:
If I remove the else it will show two alerts...

Reply:
Fair.